src/deb/control/CHANGES.txt

```diff
@@ -0,0 +1,5 @@
+[[name]] ([[version]]) stable; urgency=low
+
+  * See http://jmxtrans.googlecode.com/ for more details.
+
+ -- Pushkar Raste <praste@bloomnberg.net, pushkar.raste@gmail.com> [[buildTimeStamp]]
```

Comment:
I'm happier giving credit to people who contribute than to take credit as my own. We should keep a list of contributors.